include/swift/Remote/MetadataReader.h

```diff
@@ -524,6 +610,14 @@ class MetadataReader {
       return None;
 
     return start;
+#else
+    // Since there's no Objc interop, the InstanceSize field is at offset 3
+    auto address = RemoteAddress(MetadataAddress + (sizeof(StoredSize) * 3));
+    size_t size;
+    if (!Reader->readInteger(address, 4, &size))
```

Comment:
Does this leave the high bits of `size` uninitialized? Consider writing this as: `size_t size = 0;`.

Reply:
@vedant, this is actually outdated! I believe what we want on this function is the starting point of our fields. Let me push so you can take a look it that makes sense.